src/Pohoda/IssueSlip/Header.php

```diff
@@ -19,7 +19,7 @@ class Header extends DocumentHeader
     protected $_refElements = ['number', 'priceLevel', 'paymentType', 'centre', 'activity', 'contract', 'carrier', 'regVATinEU', 'MOSS', 'evidentiaryResourcesMOSS'];
 
     /** @var string[] */
-    protected $_elements = ['number', 'date', 'numberOrder', 'dateOrder', 'text', 'partnerIdentity', 'acc', 'symPar', 'priceLevel', 'paymentType', 'isExecuted', 'isDelivered', 'centre', 'activity', 'contract', 'carrier', 'regVATinEU', 'MOSS', 'evidentiaryResourcesMOSS', 'accountingPeriodMOSS', 'note', 'intNote'];
+    protected $_elements = ['number', 'date', 'numberOrder', 'dateOrder', 'text', 'partnerIdentity', 'acc', 'symPar', 'priceLevel', 'paymentType', 'isExecuted', 'isDelivered', 'centre', 'activity', 'contract', 'carrier', 'regVATinEU', 'MOSS', 'evidentiaryResourcesMOSS', 'accountingPeriodMOSS', 'note', 'intNote', 'histRate'];
```

Comment:
https://www.stormware.sk/xml/schema/version_2/vydejka.xsd

this says "Doklad má položky obsahující historickou sazbu DPH. **Jen pro export.**". Is it inaccurate?

Reply:
We are using it while importing the document and it works as expected. I will try to reach out to Stormware support to confirm this though.

Reply:
if you import items with historical rate, it's possible, that this works as expected, even if it's not processed